roles/openshift_descheduler/vars/openshift-enterprise.yml

```diff
@@ -0,0 +1,6 @@
+---
+# image prefix defaults
+var_openshift_descheduler_image_prefix: "{{ openshift_descheduler_image_prefix | default('registry.access.redhat.com/openshift3/ose-') }}"
```

Comment:
Don't do this.

Just put
```
openshift_descheduler_image_prefix: 'registry.access.redhta.com/openshift3/ose-'
```
in defaults/main.yml.  Same comment below.

Reply:
but there are different paths for origin and openshift-enterprise, so doing this way as default_images.yaml points to origin's default path and openshift-enterprise.yaml points to ocp's default paths? am I missing something?